solutions/04_hello3ages/hello3ages.js

```diff
@@ -0,0 +1,8 @@
+var age1 = prompt("How old are you?");
+var name1 = prompt("What is your name?");
+var age2 = prompt("How old are you?");
+var name2 = prompt("What is your name?");
+var age3 = prompt("How old are you?");
+var name3 = prompt("What is your name?");
+
+alert(name1 + "-" + age1 + "\n" + name2 + "-" + age2 + "\n" + name3 + "-" + age3);
```

Comment:
Nice use of the newline character! Can you think of a way we can do this without writing `prompt` more than twice? (hint: loops)